src/com/lftechnology/java/training/alina/collection/linkedlist/LinkedListCollection.java

```diff
@@ -0,0 +1,100 @@
+
+package com.lftechnology.java.training.alina.collection.linkedlist;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * a. Create a linked list named one and two. 
+ * b. Add some elements to both one (one, two, 
+ * three, four, five) and two (six, seven, eight, nine, and ten).
+ * c. Merge the words from two into one. 
+ * d. Remove every second word from two. 
+ * e. Remove all words in two from one. 
+ * f. Print the value after each operation.
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class LinkedListCollection {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)